### PR TITLE
Update for Fastify v5

### PR DESCRIPTION
**Pull request by synapse**
Ref: https://github.com/fastify/fastify/issues/5116

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

CC @simoneb

### CODE REVIEW

**Comment by simoneb** [comment]
done and rebased

File: .taprc [hunk 1]
```diff
@@ -1,9 +1,3 @@
-ts: false
-jsx: false
-flow: false
-jobs: 1
-coverage: true
-check-coverage: true
-
+disable-coverage: true
```
Comment (gurgunday) [inline .taprc:1]: I can't test Redis right now locally, so can you try removing this line to see if the coverage will pass?

Comment (gurgunday) [reply .taprc:1]: Before doing that, we also need to change istanbul-ignore with `/* c8 ignore next */` here:

https://github.com/fastify/fastify-redis/blob/904b2dfa25b0c3e30e192f8c15179ac98de52f7c/index.js#L70
https://github.com/fastify/fastify-redis/blob/904b2dfa25b0c3e30e192f8c15179ac98de52f7c/index.js#L91

Comment (synapse) [reply .taprc]: Done

Comment (gurgunday) [reply .taprc]:
```js
/* c8 ignore start */
  const onEnd = function (err) {
    client
      .off('ready', onReady)
      .off('error', onError)
      .off('end', onEnd)
      .quit()

    next(err)
  }
/* c8 ignore stop */
```

Comment (gurgunday) [reply .taprc]: We need to keep coverage on as much as possible

**Review by gurgunday — APPROVED** [review]
lgtm!

**Pull request merged**